### ISSUE TITLE
InputNumber with `aria-describedby` do not work as expected

### ISSUE THREAD

**Issue by lostdisplay** [primefaces/primeng#17727]
### Describe the bug

```html
<p-inputnumber aria-describedby="additional-message" />
<small id="additional-message">Additional Message</small>
```
`aria-describedby` does not get applied to the child `<input>`.
`attr.aria-describedby` also does not fix the issue.

### Reason for not contributing a PR

- [x] Lack of time
- [ ] Unsure how to implement the fix/feature
- [x] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/azje8yfg?file=src%2Fapp%2Finput-number-numerals-demo.html

### Environment

Newly created project

### Angular version

19.1 & 19.0

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Create new angular with primeng project
2. Add <p-inputnumber> with `aria-describedby` and apply it to an `<small>`-element
3. Use a screenreader like NVDA or look at the DOM
4. `aria-describedby` is not applied correctly and therefore screenreader does not read the content in `<small>`

### Expected behavior

`aria-describedby` should be applied to the underlying `<input>` child element, so that screenreaders can read the description when `p-inputnumber` is focused.
